merge/conflict_test.go

```diff
@@ -92,3 +93,44 @@ func TestToSet(t *testing.T) {
 		t.Fatalf("expected\n%v\n, but got\n%v\n", expected, actual)
 	}
 }
+
+func TestConflictsFromManagers(t *testing.T) {
+	type args struct {
+		sets fieldpath.ManagedFields
+	}
+	tests := []struct {
+		name string
+		args args
+		want string
+	}{
```

Comment:
Since you have only on test I think it's fine to remove the table-test. Also this repo generally doesn't use that format for table-tests.